src/gui/properties/speedplotview.h

```diff
@@ -29,9 +29,12 @@
 #ifndef SPEEDPLOTVIEW_H
 #define SPEEDPLOTVIEW_H
 
+#ifndef Q_MOC_RUN
```

Comment:
It not compile on Qt 4.8.6 when I remove this,
got the same error
`usr/include/boost/type_traits/detail/has_binary_operator.hp:50: Parse error at "BOOST_JOIN"`